src/components/mega-menu/mega-menu.scss

```diff
@@ -494,9 +494,14 @@ $mega-menu: null !default;
   }
 
   .ecl-mega-menu__featured-list__item-description {
-    font: map.get($mega-menu, 'global', 'featured-description-font');
     margin-inline-start: var(--s-l);
   }
+
+  .ecl-mega-menu__featured-list__item-description,
```

Comment:
it is to have higher specificity that you didn't merge this block with the previous one?